Documentation~/preferences.md

```diff
@@ -1,279 +1,109 @@
-
-![Dimensions Overlay section](images/prefs_dimover.png)
-
-By default, the overlay shows the dimensions of the selected Mesh elements only, but you can use the **Bounds Display** property to change this to show the dimensions of the select GameObjects, regardless of what elements (vertices, edges, or faces) are selected.
-
-| **Property:** | **Description:**                                             |
-| :------------ | :----------------------------------------------------------- |
-| __Object__    | Select this option to use the selected object(s) to calculate the dimensions of the world space bounds for the information that appears on the overlay. |
-| __Element__   | Select this option to use the selected element(s) to calculate the dimensions of the world space bounds for the information that appears on the overlay. This is the default. |
-
-
-
-
-
-<a name="experimental"></a>
+By default, the [Dimensions overlay](menu-dimover.md) displays the dimensions of the selected mesh elements only. To display the dimension of the entire object, regardless of the selected elements (vertices, edges, or faces), change the **Bounds Display** property from **Object** to **Element**.
 
 ## Experimental
 
-Use these settings to enable and disable ProBuilder experimental features.
-
-> **Warning:** Experimental features are untested and might break your Project.
-
-![Experimental section](images/prefs_experimental.png)
-
-| **Property:**                     | **Description:**                                             |
-| :-------------------------------- | :----------------------------------------------------------- |
-| __Experimental Features Enabled__ | Enable this option to access the [New Bezier Shape](bezier.md) experimental feature in the ProBuilder toolbar, and the __Store Mesh as Asset__ option. <br /><br />**Note:** This setting has no affect on access to the [Boolean (CSG) Tool](boolean.md), which is always available from the [Experimental menu](menu-experimental.md). |
-| __Meshes Are Assets__             | Enable this option to store Mesh information in the Project instead of in each separate Scene level. |
-
-> **Note**: When you toggle Experimental Features on or off, Unity has to recompile scripts because the changes add or remove functionality in Unity. This means that there is a delay before this option appears to change.
-
+Enable or disable ProBuilder experimental features:
 
+* **Tools** > **ProBuilder** > **Editors** > [**New Bezier Shape**](bezier.md)
+* **Tools** > **ProBuilder** > **Experimental** > **Boolean (CSG)**.
+* **Preferences** window > **ProBuilder** > **Store Mesh as Asset**.
 
+> **Warning:** Experimental features can have unpredictable results.
 
+| **Property** | **Function** |
+| :--- | :--- |
+| **Experimental Features Enabled** | Add the experimental features to the Editor. |
+| **Store Meshes as Asset** | Store meshes as standalone assets in the Project folder, rather than as part of the Scene asset. |
```

Comment:
```suggestion
| **Store Meshes as Asset** | Store meshes as standalone assets in the Project folder, rather than as part of the Scene asset. This property is only visible when **Experimental Features Enabled** is enabled. |
```